felles/src/main/kotlin/no/nav/emottak/vault/VaultUtil.kt

```diff
@@ -79,7 +79,9 @@ class VaultUtil private constructor() {
         private val instance: VaultUtil = VaultUtil()
 
         fun readVaultPathResource(path: String, resource: String): String =
-            instance.client.logical().read(path).data[resource] ?: throw RuntimeException("Failed to read vault path resource: $path/$resource")
+            instance.client.logical().read(path).data[resource].also {
+                logger.info("Got vault resource $resource from vault path $path")
```

Comment:
Vi blander `log` og `logger` flere steder. Burde vi bestemme oss for en standard vi bør følge?